src/helpers.js

```diff
@@ -144,7 +144,7 @@ export function normalizeSwagger(parsedSpec) {
           toBeInherit.parameters = pathParameters
           inheritsList.push(toBeInherit)
         }
-
+        
```

Comment:
@saharj get `editorconfig` plugin for you IDE... it'll help remove these little line things.